bower.json

```diff
@@ -26,7 +26,7 @@
 	],
 	"dependencies": {
 		"angular":">=1.3.0",
-		"sweetalert":"^1.0.0"

```

Comment:
Aren't these matchers functionally equivalent?

https://stackoverflow.com/a/22345808/1466565

Reply:
Not really, ^1.0.0 will still look up for major version 2 which include major changes.
indicating 1.^ will get the latest release on version 1.

Reply:
The documentation I've read says that ^1.0.0 supports all 1.x.x releases. Admittedly, it's pretty old, so if you see something different, please let me know.

I'd suggest that the library only supports versions that we explicitly know work. So since we know 1.1.3 works, we just set that as the dependency. That way if 1.2.0 or 1.1.4 is released and breaks something, this library doesn't break as well.

Reply:
It would be great to release a new version of ngSweetAlert that contains the changes of #79.

Reply:
Any updates on this PR.